.github/workflows/ci.yaml

```diff
@@ -235,6 +235,43 @@ jobs:
           echo "skip_coverage: ${skip_coverage}"
           echo "skip_coverage=${skip_coverage}" >> $GITHUB_OUTPUT
 
+  sync-github-labels:
+    name: Sync GitHub integration labels
+    runs-on: ubuntu-24.04
+    if: github.ref == 'refs/heads/dev'
+    needs:
+      - info
+    steps:
+      - name: Check out code from GitHub
+        uses: actions/checkout@v4.2.2
+      - name: Sync integration labels
+        run: |
+          gh_labels=$(gh api /repos/${OWNER}/${REPO}/labels \
+            --paginate \
+            --jq '.[] | select(.name | startswith("integration: ")) | "\(.name)|\(.description);"')
```

Comment:
Shouldn't we only run this when names change?

Reply:
valid point - i think we can limit the task to run only when any `manifest.json` has been changed :thinking:

Reply:
But shouldn't this be a separate job that we run on the `dev` branch instead of in the CI?

Reply:
sure, we can also split it into an own workflow. would maybe also makes it easier :thinking: